containers/profile/info.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <section class="quick-info padded">
+    <h1 class="btn-align vertical">{{ user.name }} {{ user.surname }}</h1>
+    <div class="balance">
+      <p class="btn-align">
+        <span>{{ user.innopoints }}</span>
+        <img src="/images/innopoint.svg" />
+        available
+      </p>
+      <nav class="btn-shift">
+        <Button
+          link
+          href="/shop"
+          img="/images/profile/shopping-bag.svg"
+          label="go to the InnoStore"
+          class="mr"
+        />
+        <Button
+          link
+          href="#"
+          img="/images/profile/medal.svg"
+          label="request innopoints"
+          class="mr"
+        />
+      </nav>
+    </div>
+
+    <div class="personal">
+      <h1 class="horizontal">{{ user.name }} {{ user.surname }}</h1>
+      <div class="telegram">
+        <Button
+          outline
+          img="/images/profile/send.svg"
+          label="add Telegram username"
+        />
+
+        <Dropdown :chevron="false" right>
+          <template v-slot:opener>
+            <img src="/images/profile/help-circle.svg" class="icon" />
+          </template>
+          <template>
```

Comment:
No need to wrap the default slot in `<template>` tags

Reply:
Done